go/protocol/keybase1/simple_fs.go

```diff
@@ -661,6 +661,18 @@ func (o OpProgress) DeepCopy() OpProgress {
 	}
 }
 
+type SimpleFSHTTPServerAddressAndToken struct {
```

Comment:
This name is unreadable. Can you make it something simpler?

Reply:
Sorry I changed what we agreed on because it's now in KBFS, and that we needed address in addition to just the token.

How about `SimpleFSGetHTTPAddressAndToken`?

Reply:
Yeah that's better, thanks!